test/SlowTests/Client/TimeSeries/BulkInsert.cs

```diff
@@ -1212,5 +1219,88 @@ void AssertError()
                 }
             }
         }
+        
+         [Fact]
```

Comment:
wrong indent

Reply:
don;t we already have test for that ?

Reply:
1.  fixed indentation
2.  I didn't see a test with bulk insert for all 4 types, so I added